.github/workflows/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.yml]
```

Comment:
Can't we just put it in 1 central editorconfig file?

Reply:
But I only want to to effect the workflow files. 

Im not sure I can do that from the root editorconfig.

Reply:
If im correct you can just add it and the IDE will pick it up from the app docroot and parse all files like that. Can you give it a try?

Reply:
Yeah, But I dont want to apply this rule to ALL yml files. Only yml files in `.github/workflows`